in_toto/util.py

```diff
@@ -134,8 +149,39 @@ def prompt_password(prompt="Enter password: "):
   return getpass.getpass(prompt, sys.stderr)
 
 
+def prompt_import_private_key_from_file(filepath):
+  """Trys to load an Ed25519 or RSA key without password. If a CryptoError
+  occurs, prompts the user for a password and trys to load the the key again.
+  """
+  # FIXME: The more secure method is to load the private key using the key type
```

Comment:
This function is dangerous in the sense that it makes the assumption that a private key is always either an Ed25519, or an RSA key, in that order. I haven't fully thought this through, but an attacker might be able to exploit this in order to bind private keys to [mismatching cryptographic schemes](https://www.nccgroup.trust/us/our-research/docker-notary/).

A more robust function would either: (1) ask the user to explicitly identify the keytype, and / or (2) correctly guess the keytype from information in the private key itself.